packages/playground/src/components/openapi-output.tsx

```diff
@@ -12,7 +12,7 @@ export interface OpenAPIOutputProps {
 export const OpenAPIOutput: FunctionComponent<OpenAPIOutputProps> = (props) => {
   const [selected, setSelected] = useState<"raw" | "swagger-ui">("raw");
   const options = [
-    { label: "OpenAPI", value: "raw" },
+    { label: "Output Files", value: "raw" },
```

Comment:
hhm, thats another issue didn' think of, we don't really want that dropdown to be showing for non openapi output

Reply:
Yeah, you get a kind of ugly warning about missing the OpenAPI version number if you set it to Swagger UI when the protobuf emitter is selected...

Reply:
maybe we should have yet another config to load a plugin(swagger view) for certain emitters. become out of scope of your pr though